HelloMixpanel/HelloMixpanel/ViewController.m

```diff
@@ -85,7 +79,7 @@ - (IBAction)setPeopleProperties:(id)sender
 
 - (IBAction)setNotificationType:(id)sender
 {
-    NSArray *types = @[MPNotificationTypeTakeover, MPNotificationTypeMini];
+    NSArray *types = @[@"takeover", @"mini"];
```

Comment:
The aliases are not exposed to the users of the framework. These strings came from the `Mixpanel.h` header.